File: src/test/java/no/nav/syfo/consumer/util/ws/WsClientTest.java
package no.nav.syfo.consumer.util.ws;

import no.nav.syfo.consumer.ws.util.LogErrorHandler;
import no.nav.syfo.consumer.ws.util.WsClient;
import no.nhn.schemas.reg.flr.IFlrReadOperations;
import org.junit.Test;

import static java.util.Collections.singletonList;

public class WsClientTest {
    static final String PORT_URL =  "http://url.dev";

    @Test
    public void testsomething() {

        IFlrReadOperations port = new WsClient<IFlrReadOperations>()
                .createPort(PORT_URL, IFlrReadOperations.class, singletonList(new LogErrorHandler()));
    }
}

File: src/test/kotlin/no/nav/syfo/consumer/ws/util/WsClientTest.kt
package no.nav.syfo.consumer.ws.util

import no.nhn.schemas.reg.flr.IFlrReadOperations
import org.junit.Test

class WsClientTest {
    @Test
    fun testsomething() {
        val port = WsClient<IFlrReadOperations>()
            .createPort(PORT_URL, IFlrReadOperations::class.java, listOf(LogErrorHandler()))
    }

    companion object {
        const val PORT_URL = "http://url.dev"
    }
}
